src/server/data/mapping/filterValidOppfolgingsplaner.ts

```diff
@@ -0,0 +1,126 @@
+import { Oppfolgingsplan } from "../../../schema/oppfolgingsplanSchema";
+import {
+  DineSykmeldteSykmelding,
+  Sykmeldt,
+} from "../../../schema/sykmeldtSchema";
+import { differenceInDays } from "date-fns";
+
+export const MND_SIDEN_SYKMELDING_GRENSE_FOR_OPPFOLGING = 4;
+
+interface FravaerPeriode {
+  fom: Date;
+  tom: Date;
+}
+
+const mapToPerioderDatesListSorted = (sykmelding: DineSykmeldteSykmelding) => {
+  return sykmelding.mulighetForArbeid.perioder
+    .map((periode) => {
+      return { fom: new Date(periode.fom), tom: new Date(periode.tom) };
+    })
+    .sort(compareFomDate);
+};
+
+function compareFomDate(a: FravaerPeriode, b: FravaerPeriode) {
```

Comment:
Kanskje vi kan bruke `date-fns` for å sammenligne?

Reply:
Sa du at denne filen var kopiert?

Reply:
Kopiert funksjonene, slått sammen i en fil :)

Reply:
Men ja, ser ut som de har en funksjon compareDesc som gjør det samme:) Endrer